### ISSUE TITLE
FileNotFoundError

### ISSUE THREAD

**Issue by Gnusmas49** [lagmoellertim/unsilence#13]
When I try to use unsilence, the following happens:

```
Rendering Intervals... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
Combining Intervals... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0% -:--:--
FileNotFoundError: [WinError 2] 系统找不到指定的文件。: 'C:\\Users\\h-zhao_01\\.tmp\\2815269f-cf00-4535-bce9-7c0523be2a54\\out_final.mp4' -> 'D:\\videos\\output.mp4'

During handling of the above exception, another exception occurred:

FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\h-zhao_01\\.tmp\\2815269f-cf00-4535-bce9-7c0523be2a54\\out_final.mp4'
```

I have tried with ffmpeg 4.3 and 4.2.3 versions, and both of them produced the same error.  It seems that this is a different issue from #4.  Is there anything I can do to fix it?  Thank you!

**Comment by lagmoellertim** [comment]
Hey, I'm sorry I haven't answered till now, I'm really short on time and probably am not able to look any further into your Issue until next month, sorry!

**Comment by AtilioA** [comment]
@Gnusmas49 PR #8 might have fixed this, could you [reinstall unsilence from source](https://github.com/lagmoellertim/unsilence#installation-as-command-line-interface-from-source) and give us some feedback?

**Comment by lagmoellertim** [comment]
I'm closing this for now since I have reason to believe that this issue is fixed with PR #8 .
If this issue still  persists, feel free to reopen this issue!

**Comment by TheCidRMC** [comment]
hi I have the same problem even when I do this in line command (Win 10) and run this: 
`unsilence a.wav b.wav`

what can I do?
